Fails to run samples since the last ROS sync

**Describe the Bug**
Since the last ROS sync, the Webots examples cannot be run and therefore custom-built simulations are failing as well.

**Steps to Reproduce**

```
$ ros2 launch webots_ros2_turtlebot robot_launch.py 
[INFO] [launch]: All log files can be found below /home/lukic/.ros/log/2022-12-18-11-56-38-546036-ltp-51088
[INFO] [launch]: Default logging verbosity is set to INFO
WARNING: No valid Webots directory specified in `ROS2_WEBOTS_HOME` and `WEBOTS_HOME`, fallback to default installation folder /usr/local/webots.
[INFO] [webots-1]: process started with pid [51089]
[INFO] [ros2_supervisor.py-2]: process started with pid [51091]
[INFO] [spawner-3]: process started with pid [51093]
[INFO] [spawner-4]: process started with pid [51097]
[INFO] [robot_state_publisher-5]: process started with pid [51099]
[INFO] [driver-6]: process started with pid [51101]
[ERROR] [driver-6]: process has died [pid 51101, exit code 127, cmd '/opt/ros/humble/lib/webots_ros2_driver/driver --ros-args --params-file /tmp/launch_params_sauvegp4 --params-file /opt/ros/humble/share/webots_ros2_turtlebot/resource/ros2control.yml -r /diffdrive_controller/cmd_vel_unstamped:=/cmd_vel -r /diffdrive_controller/odom:=/odom'].
[INFO] [static_transform_publisher-7]: process started with pid [51104]
[driver-6] /opt/ros/humble/lib/webots_ros2_driver/driver: error while loading shared libraries: libController.so: cannot open shared object file: No such file or directory
...
```

**System**
 - Webots Version: R2023a
 - ROS Version: Humble
 - Operating System: Ubuntu 22.04

**Additional context**
For anyone encountering the same problem a quick fix can be manually adding missing paths, e.g.:
```py
os.environ['LD_LIBRARY_PATH'] += ':/opt/ros/humble/lib/controller'
```

--- a/webots_ros2_driver/CMakeLists.txt
+++ b/webots_ros2_driver/CMakeLists.txt
@@ -181,7 +181,7 @@ install(
 
 # Create symlinks of libController
 macro(lib_symlink filename)
-  install(CODE "execute_process(COMMAND ln -sf ${CMAKE_INSTALL_PREFIX}/lib/controller/${filename} ${CMAKE_INSTALL_PREFIX}/lib/${filename})")
+  install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_INSTALL_PREFIX}/lib/controller/${filename} ${CMAKE_INSTALL_PREFIX}/lib/${filename})")
 endmacro(lib_symlink)
 
 lib_symlink(libController.so)

